cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -14,6 +14,7 @@ kind: Namespace
 metadata:
   name: rook-ceph
 ---
+# TAG: BEGIN CEPH CRD
```

Comment:
What about `OLM` instead of `TAG`? Then it's more obvious why the need for these tags.

Reply:
The tags are kind of annoying for users. If we don't use all caps maybe it will help? For example: `OLM: Begin Ceph CRD`. And perhaps we should have a description at the top of the yaml about why the OLM tags are needed.

Reply:
I changed to use OLM but kept the capital letters, also added a section to explain what they are.